target/linux/ath79/image/generic.mk

```diff
@@ -1683,6 +1683,16 @@ define Device/yuncore_xd4200
 endef
 TARGET_DEVICES += yuncore_xd4200
 
+define Device/ziking_cpe46b
+  SOC := ar9330
+  DEVICE_VENDOR := ZiKing
+  DEVICE_MODEL := CPE46B
+  IMAGE_SIZE := 8000k
+  SUPPORTED_DEVICES += cpe46b
```

Comment:
Is this for vendor firmware?

Reply:
Do you mean the image size? It is actually the net of 8192k-64k(uboot)-64k(u-boot-env)-64k(art)

Reply:
No, I mean SUPPORTED_DEVICES

Reply:
You're right, I'll drop the line

Reply:
How can I be right when I ask a question?

Reply:
You're right that that line is there for no other purpose than a leftover while copying from another target. Are the variables there documented anywhere?

Reply:
So, with the line dropped, I suspect one will need to force upgrade now coming from the vendor firmware?

Reply:
I'm getting a secondary device with stock firmware in order to test installing openwrt directly from the vendor updater